src/CMakeLists.txt

```diff
@@ -89,7 +89,7 @@ if(USE_ROCM)
 endif()
 
 if(USE_CUDA)
-  set_target_properties(sirius PROPERTIES CUDA_ARCHITECTURES ${CMAKE_CUDA_ARCHITECTURES})
+  set_target_properties(sirius PROPERTIES CUDA_ARCHITECTURES ${CMAKE_CUDA_ARCHITECTURES} CUDA_RESOLVE_DEVICE_SYMBOLS ON)
```

Comment:
Wo we even need to do this line? According to CMkae doc. CMAKE_CUDA_ARCHITECTURES provides a default for CUDA_ARCHITECTURES.

Reply:
CUDA_ARCHITECTURES can be removed that's not the issue.